### PR TITLE
Apply vtex button

### PR DESCRIPTION
**Pull request by augustolazaro**
#### What is the purpose of this pull request?
Use VTEX styleguide even more.

#### What problem is this solving?
Related to https://app.clubhouse.io/vtex/story/34375/migrar-bot%C3%B5es-do-admin-para-o-styleguide-vtex

#### How should this be manually tested?
https://augusto--mystore.mygocommerce.com/admin/logistics/gocommerce.customshipping-app/

#### Screenshots or example usage
Screen Shot 2020-03-24 at 18 01 29

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/carriers/list/components/ButtonDeleteCarrier.tsx [hunk 1]
```diff
@@ -53,61 +54,55 @@ class ButtonDeleteCarrier extends React.Component<ButtonDeleteCarrierProps, Butt
...
+              <Modal open={isModalOpen} onClose={this.handleClick} showCloseIcon={true} centered={false}>
+                <div>
+                  <p className="g-ma0 g-f4 fw6">
+                    <FormattedMessage id="admin/shipping.modal-delete-carrier-title" />
+                  </p>
+                  <p className="g-mv5 g-f2 lh-copy">
+                    <FormattedMessage id="admin/shipping.modal-delete-carrier-description" />
+                  </p>
+                  <div className="flex flex-none-ns justify-between justify-end-ns g-mt8 g-nh3">
+                    <div className="pointer g-mh3"><Button
```
Comment (gustavorosolem) [inline react/carriers/list/components/ButtonDeleteCarrier.tsx]: Acho que pode remover o pointer

File: react/carriers/list/components/ButtonDeleteCarrier.tsx [hunk 2]
```diff
@@ -53,61 +54,55 @@ class ButtonDeleteCarrier extends React.Component<ButtonDeleteCarrierProps, Butt
...
-                        onClick={this.handleClick}
-                      >
+                        variation="secondary"
+                        onClick={this.handleClick}>
                         <FormattedMessage id="admin/shipping.modal-delete-carrier-no" />
-                      </Button>
+                      </Button></div>
 
-                      <Button
+                    <div className="pointer g-mh3"><Button
```
Comment (gustavorosolem) [inline react/carriers/list/components/ButtonDeleteCarrier.tsx]: Same

File: react/carriers/list/components/ButtonDeleteCarrier.tsx [hunk 3]
```diff
@@ -53,61 +54,55 @@ class ButtonDeleteCarrier extends React.Component<ButtonDeleteCarrierProps, Butt
...
                         size="large"
-                        style="danger"
-                        className="pointer g-mh3"
+                        variation="danger"
                         onClick={() => {
                           this.handleClickYes(deleteCarrier)
                         }}
-                        isDisabled={false}
-                      >
+                        isDisabled={false}>
```
Comment (gustavorosolem) [inline react/carriers/list/components/ButtonDeleteCarrier.tsx]: Na vtex o atributo é apenas `disabled `

Comment (augustolazaro) [reply react/carriers/list/components/ButtonDeleteCarrier.tsx]: Na GC também, por isso o script não pegou esse caso 🤦‍♂

**Pull request merged**